benches/shrink_wrap.rs

```diff
@@ -6,7 +6,7 @@ use criterion::{black_box, criterion_group, criterion_main, BenchmarkId, Criteri
 
 fn bench_shrink_wrap(c: &mut Criterion) {
     let mut group = c.benchmark_group("shrink_wrap");
-    let geometry = geometry::uv_sphere([0.0, 0.0, 0.0], 1.0, 3, 3);
+    let geometry = geometry::uv_sphere_geometry([0.0, 0.0, 0.0], 1.0, 3, 3);
```

Comment:
it now follows the convention I've started before that a generator returning a geometry is suffixed with `_geometry`. the reason to do it now was a new data type `Plane`, which was in conflict with the original `plane` generator.
leaving as is, the refactor will follow

Reply:
Yes, I know your reasoning, just disagree with it :joy: 

What I was suggesting is that because you wanted to move the `Plane` and related logic to it's own file (IIRC), there would no longer be a naming conflict and no reason to rename them. I don't think we should rename the primitives, because the subsequent cleanup PR will move them to their own file and we will be calling them throughout the project this way: `mesh::primitive::sphere(...)`, so the geometry suffix won't make sense anymore.

Reply:
If you really want to, we can do that work in the cleanup PR and not now.

Reply:
oh, a disagreement! good.
i don't think this is only about the naming conflict. as i mentioned above, there are some generators that return `Geometry` and some that only return geometry components and you need to create the `Geometry` yourself. therefore this remains relevant even after the refactor.
nevertheless, shouldn't the refactor be a separate task that happens some time in the future and if we decide the suffix is not needed anymore, we remove it there? although I thin we should keep it even there

Reply:
Ok, I don't have very strong feelings about this, so let's not spend more time on this now.

Reply:
no vidíš ;)